packages/@aws-cdk/custom-resources/README.md

```diff
@@ -100,7 +100,7 @@ def is_complete(event, context):
   return { 'IsComplete': is_ready }
 ```
 
-[custom resources]: (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-custom-resources.html).
+[custom resources]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-custom-resources.html
```

Comment:
seems like this is only used in one location - why not just move it inline where it's clearer what it links to?